tools/expotools/yarn.lock

```diff
@@ -9129,6 +9141,15 @@ shell-quote@1.6.1:
     array-reduce "~0.0.0"
     jsonify "~0.0.0"
 
+shelljs@^0.8.3:
```

Comment:
`shelljs` had some CVE recently, maybe we could just not use it?

Reply:
That's definitely a goal, I replaced some invocations of `shelljs` with `spawn-async`, but mostly because they were failing (probably due to `cwd` being set incorrectly). I'll be refactoring it later, after release.